web/src/features/charts/graphUtils.test.ts

```diff
@@ -140,3 +141,42 @@ describe('getTotalEmissionsAvailableOrElectricityAvailable', () => {
     expect(actual).to.deep.eq(Number.NaN);
   });
 });
+
+describe('extractLinkFromSource', () => {
+  const sourceLinkMapping = {
+    source1: 'http://mappedlink1.com',
+    source2: 'http://mappedlink2.com',
+    Climatescope: 'https://www.global-climatescope.org/',
+  };
+
+  it('should return mapped link if source is in sourceLinkMapping', () => {
+    expect(extractLinkFromSource('source1', sourceLinkMapping)).to.equal(
+      'http://mappedlink1.com'
+    );
+    expect(extractLinkFromSource('source2', sourceLinkMapping)).to.equal(
+      'http://mappedlink2.com'
+    );
+  });
+
+  it('real example: should return mapped link if source is in sourceLinkMapping', () => {
+    expect(extractLinkFromSource('Climatescope', sourceLinkMapping)).to.equal(
+      'https://www.global-climatescope.org/'
+    );
+  });
```

Comment:
Dunno how much it matters, but if we stick to a consistent naming it could be something like this:
```suggestion
  it('should work with a real link', () => {
    expect(extractLinkFromSource('Climatescope', sourceLinkMapping)).to.equal(
      'https://www.global-climatescope.org/'
    );
  });
```